### DISCUSSION TITLE
Ограничения границ отображения пвз

### DISCUSSION THREAD

**Discussion by Andrew100500** [cdek-it/widget#65] [Q&A] [answered]
Я хочу отображать только ПВЗ которые находятся в заданном городе, мне не нужно показывать их все для всей страны, чтобы покупатели по ошибке не туда тыкали. 
Параметру fixBounds задаем значение locality  и получаем в каких то городах список ПВЗ, в каких то фигу с маслом.  Например Нальчик, при данных установках в нем оказывается нет ПВЗ. Я еще в феврале об этом вам намисал. Вы ответили что это будет исправлено в ближайшем обновлении. И как обычно ничего не исправлено.

**Comment by vermorag** [comment]
Добрый день. Подскажите номер вашего обращения.

**Comment by Andrew100500** [comment]
SDPO-74384  Вы замечательно пишите исключительно отмазки.  Вместо исправления ошибок Вы их отстаиваете с завидным упорством. В том обращении была проблема с  Карачаево-Черкесской Республикой. Но на самом деле у вас проблемных регионов скорее всего дохрена. Я еще штук 5-6 городов нашел,  в которых нет ПВЗ если fixBounds значение locality

**Reply by vermorag** [reply, marked as answer]
Как уже сообщалось в вашем обращении, до момента выпуска 4 версии виджета (запланирована к выпуску до конца II квартала 2024 года) повлиять на подобное поведение параметра fixBounds возможности нет, поскольку он использует БД яндекса для определения населенных пунктов и регионов, которая не совпадает с БД СДЭК-а (а также не совпадает с БД Почты РФ, которую вы приводили в пример).
Поэтому рекоммендуется использовать ограничения для области (значение `province`).

**Comment by Andrew100500** [comment]
Еще кстати, не цветная карта - хреновое улучшение.

**Reply by vermorag** [reply]
Виджет ПВЗ является базовым универсальным решением. Вы можете видоизменять его внешний вид самостоятельно с помощью скриптов/настроек яндекс карты/css стилей.

**Comment by Andrew100500** [comment]
Ага и Казань тоже неправильно определяется. А пацаны то и не знают. Может Казань стоит переименовать, ну чтобы она с вашей базой сходилась?

**Reply by vermorag** [reply]
По данному вопросу вам необходимо обратиться на integrator@cdek.ru с описанием вашего запроса.

**Comment by Andrew100500** [comment]
И ограничение для области тоже не срабатывает.

**Reply by vermorag** [reply]
Полнофункционально ограничение будет работать с 4 версии.

**Comment by Andrew100500** [comment]
Хотя понимаю, что многого от вас хочу.  Вы до сих пор Е на Ё не никак исправите.

**Comment by Andrew100500** [comment]
> По данному вопросу вам необходимо обратиться на [integrator@cdek.ru](mailto:integrator@cdek.ru) с описанием вашего запроса.

Мне нужно куда-то писать и просить чтобы вы исправили свою собственную ошибку. Серьезно? Это ваша ошибка, это ваша проблема.

**Comment by Andrew100500** [comment]
Господа, Вы долго еще сайтые вещать будете своим дырявым сервисом?

**Comment by Andrew100500** [comment]
Хотя какие господа. Такое ощущение, что Эйнштейн на доске код написал и вышел. А остальное стадо программистов стоит и смотрит на какаркули написанные на неизвестном для них языке.

**Comment by Andrew100500** [comment]
Ну так как дела с 4 версией. 2 квартал давно закончился, уже можно сказать треться часть червертого кварата прошла. Где 4 версия? 
Вам не один я пишу, что не нужно человеку из мухостранска показывать ПВЗ в Гваделупе.  А если примениь locality  то окажетсмя что в Мухосранске нет ПВЗ.   
Для крупной компании по доставке, решение таких проблем должно исчисляться минутами, а не годами.
